scvi/inference/posterior.py

```diff
@@ -31,7 +32,7 @@ def __iter__(self):
 
 class Posterior:
     r"""The functional data unit. A `Posterior` instance is instanciated with a model and a gene_dataset, and
-    as well as additional arguments that for Pytorch's `DataLoader`. A subset of indices can be specified, for
+    as well as additional arguments that for Pytorch"s `DataLoader`. A subset of indices can be specified, for
```

Comment:
Did you mean to replace this apostrophe with a quote mark?

Reply:
Misuse of ctrl-replace I guess